stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/internal/MessageMapper.kt

```diff
@@ -89,7 +90,8 @@ internal fun Message.toEntity(): MessageEntity = MessageEntity(
         reactionCounts = reactionCounts,
         reactionScores = reactionScores,
         shadowed = shadowed,
-        mentionedUsersId = mentionedUsers.map(User::id),
+        mentionedUsers = mentionedUsers.map(User::id),
+        mentionedUsersId = mentionedUsersIds,
```

Comment:
Hey @adasiewiczr,

I've tested your patch and it works as expected.
However I wonder if we can do something like this:
```kotlin
mentionedUsersId = when (mentionedUsers.isNotEmpty()) {
    true -> mentionedUsers.map(User::id)
    else -> mentionedUsersIds
}
```

To avoid having in DB a more or less duplicate column.

Reply:
Makes sense.
Do you think it's possible to rename `mentionedUsers` to something `remoteMentionedUserIds`?

The first time you look at `mentionedUsers` you would expect to see `users` and not `userIds` again.